plugin/build.gradle.kts

```diff
@@ -51,6 +51,29 @@ publishing {
     }
 }
 
+/**
+ * Configures the publishing environment for publishing with Travis CI.
+ * All you need to do is push a tagged commit to github and Travis CI will automatically publish a
+ * release of the plugin using the current [Project.getVersion].
+ */
+fun setupPublishingEnvironment() {
+    val keyEnvironmentVariable = "GRADLE_PUBLISH_KEY"
+    val secretEnvironmentVariable = "GRADLE_PUBLISH_SECRET"
+
+    val keyProperty = "gradle.publish.key"
+    val secretProperty = "gradle.publish.secret"
+
+    if (System.getProperty(keyProperty) == null || System.getProperty(secretProperty) == null) {
+        logger
+            .info("`$keyProperty` or `$secretProperty` were not set. Attempting to configure from environment variables")
+
+        System.setProperty(keyProperty, System.getProperty(keyEnvironmentVariable))
```

Comment:
This is the problem on CI:
```kotlin
Caused by: java.lang.NullPointerException
        at Build_gradle.setupPublishingEnvironment(build.gradle.kts:70)
        at Build_gradle.<init>(build.gradle.kts:75)
```

Reply:
This is caused when `GRADLE_PUBLISH_KEY` or `GRADLE_PUBLISH_SECRET` is unset.

Reply:
Thanks for figuring out where this was coming from. I haven't had time to fix this.
Feel free to resolve this if you have the time. Otherwise, I'll try to get to this later this week.